src/elements/a-gltf-entity.js

```diff
@@ -83,6 +102,20 @@ const inflateEntities = function(classPrefix, parentEl, node) {
 
   el.setObject3D(node.type.toLowerCase(), node);
 
+  const entityComponents = node.userData;
+
+  if (entityComponents) {
+    for (const prop in entityComponents) {
+      if (entityComponents.hasOwnProperty(prop)) {
+        const { deserializer, componentName } = AFRAME.AGLTFEntity.components[prop];
+
+        if (deserializer) {
+          deserializer(el, componentName, entityComponents[prop]);
```

Comment:
Not a very strong preference, but this usage makes me prefer the term "inflator' just because I expect a "deserializer" function to return a value.